lib/teiserver/bridge/discord_system.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Teiserver.Bridge.DiscordSystem do
+  use DynamicSupervisor
+  use Task
+
+  def start_link(_) do
+    DynamicSupervisor.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  @impl true
+  def init(_) do
+    pid = DynamicSupervisor.init(strategy: :one_for_one)
```

Comment:
the variable name is misleading, this is not a `pid`, but `{:ok, sup_flags()}` as per [the doc](https://hexdocs.pm/elixir/DynamicSupervisor.html#init/1)